OutOfSchool/OutOfSchool.DataAccess/Models/Configurations/ContactConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.Json;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace OutOfSchool.Services.Models.Configurations;
+
+internal class ContactConfiguration : IEntityTypeConfiguration<Contact>
+{
+    public void Configure(EntityTypeBuilder<Contact> builder)
+    {
+        builder.HasKey(x => x.Id);
+
+        builder.Property(e => e.OptionalContacts)
+                .HasConversion(
+                    v => JsonSerializer.Serialize(v, (JsonSerializerOptions)null),
```

Comment:
JsonSerializer has Serialize/Deserialize methods with null options, I use implicitly typed (JsonSerializerOptions)null to avoid the “Invocation tree...” error.

Reply:
Ok, can we fix it in any other way instead of this ?
if it is impossible - could you please add a comment that will explain why this code is here

Reply:
I changed the part to JsonConvert, so we don't need these “magic” options